cfpb/capital-framework
# Use CFPB brand colors in cf-forms component

Adds CFPB brand colors as the default variable values for colors in `cf-forms`. Includes fixes to bring this up to date with the latest Design Manual spec - see GHE for details: https://GHE/CFGOV/platform/issues/1085

Depends on #561 - this includes changes from that branch in the diff.

## Additions

- Adds large target checkboxes and radio buttons components to Capital Framework documentation 


## Changes

- Rename `@input-bg__active` to `@input-bg__selected` to match existing naming conventions
- fixes typo in class name for input states example in documentation
- updates form variables to match latest Design Manual specs (see example in @nataliafitzgerald's fork of where the DM is currently heading - her PR is in the works: https://nataliafitzgerald.github.io/design-manual/page-components/form-fields.html#labels-and-helper-text)

## Testing

In this repo:
- check out this branch
- run `npm run cf-link`

In a second local clone of this repo in a different folder:

- pull down and check out the `gh-pages-rm-overrides` branch: `git fetch && git checkout gh-pages-rm-overrides`
- if you haven't already in this second clone, [run all the install steps for the project](https://github.com/cfpb/capital-framework/tree/gh-pages#installation)
- run `npm run cf-link`
- run `npm run build`
- run `npm start`
- visit http://localhost:3000/components/cf-forms/ 
- Compare colors on local site: color values and styles there should match the documentation in [this Design Manual fork](https://nataliafitzgerald.github.io/design-manual/page-components/form-fields.html).


## Screenshots

### Patterns added/updated in docs- 

#### large target inputs
![screen shot 2017-08-22 at 6 28 03 pm](https://user-images.githubusercontent.com/702526/29590434-c9f45168-8767-11e7-8f2e-5e43c07a933b.png)

height is now 50 pixels matching design manual spec (plus borders makes it 52px)
![screen shot 2017-08-22 at 6 28 24 pm](https://user-images.githubusercontent.com/702526/29590435-c9f69108-8767-11e7-82a1-9adf6b664a9f.png)


#### helper text
![screen shot 2017-08-22 at 6 04 19 pm](https://user-images.githubusercontent.com/702526/29589811-be0106b0-8764-11e7-80e1-1a8ec5d89a39.png)


## Notes

-

## Todos

-

## Checklist
